### DISCUSSION TITLE
New firmware update error (jyers 2.0.0.)

### DISCUSSION THREAD

**Discussion by bohemtucsok** [Jyers/Marlin#1526] [Q&A]
Hi

 I had a little problem with the new firmware, I can't update the printer I don't notice the firmware file, Printer ender 3 v2 4.2.2 The process:
SD card deletion new firmware copy printer will turn off then back, but the update will not go down I will get back the 1.3.5 firmware. What would I guess wrong ???

**Comment by kted** [comment]
Make sure you are using a freshly formatted (FAT32) SD card, and the firmware's filename is different from the one you tried to flash last.
Turn the printer off, insert the card and then turn the printer on.

**Reply by bohemtucsok** [reply]
I reformatted the sd card to fat32, renamed the firmware file and booted and it doesn't change that much

**Reply by kted** [reply]
I can't  think of anything else than trying with a different SD card. Just make sure it is not larger than 32GB. Use the one that came with the printer if you don't mind loosing the presliced samples...

**Reply by bohemtucsok** [reply]
I tried with several cards and even with a new card :( I'm completely clueless, it loads the current firware on it and that's it :(

**Reply by bohemtucsok** [reply]
got the solution :) ready joke, windows formatted the card fat32 away the MBR record disappeared. I reformatted the card and gave it an mbr record, copied the new firmware and the printer was successfully updated. Thanks for the help
